modules/apps/feature-flag/feature-flag-test/src/testFunctional/paths/FeatureFlag.path

```diff
@@ -12,7 +12,7 @@
 <tbody>
 	<tr>
 		<td>TOGGLE_ROW</td>
-		<td>//*[@class="list-group-title"][contains(text(),'${title_row}')]//ancestor::ul//label//span[contains(@class,'toggle-switch-label')][contains(text(),'${toggle_state}')]</td>
+		<td>//li[(contains(.,'${title_row}')) or (contains(@for,'${title_row}'))]//span[text()='${toggle_state}']</td>
```

Comment:
I updated the path to work the same way.

Reply:
[(contains(.,'${title_row}')) or (contains(@for,'${title_row}'))]

What's the difference between these?

Reply:
https://testray.liferay.com/reports/production/logs/2023-10/test-1-31/test-portal-acceptance-pullrequest(master)/9759/functional-tomcat90-mysql57-chrome100-jdk8/30/14/LocalFile.FeatureFlag_CanBeEnabled/index.html.gz?authuser=0

The other tests pass with the old xpath. I would prefer the old xpath than the new one since the class attributes help describe the toggle better.